src/theme/general.css

```diff
@@ -50,3 +51,8 @@
   --background: #b81313;
   --color: #ffffff;
 }
+
+.DiscussionCard {
+  text-align: center;
+  width: 100%;
+}
```

Comment:
.DiscussionCard ändert hier den Style nicht. Die width der Components ist kein CSS problem, sondern falsches Wrapping von Ionic Components. Wir können uns das gerne mal anschauen. Außerdem sollte das CSS was bestimmte Components betrifft nicht in general.css liegen sondern in den CSS files für den bestimmten Component oder die Page.